packages/framework/synthesize/src/test/dependencyContainer.spec.ts

```diff
@@ -44,20 +42,21 @@ class MockLoadable implements IFluidLoadable {
 	}
 }
 
-class MockFluidRouter implements IFluidRouter {
-	public get IFluidRouter() {
+const ISomeObject: keyof IProvideSomeObject = "ISomeObject";
+interface IProvideSomeObject {
+	readonly ISomeObject: ISomeObject;
+}
+interface ISomeObject extends IProvideSomeObject {
+	value: number;
+}
+class MockSomeObject implements ISomeObject {
+	public get ISomeObject() {
 		return this;
 	}
-	public async request() {
-		return {
-			mimeType: "",
-			status: 200,
-			value: "",
-		};
-	}
+	public readonly value = 0;
 }
 
-describe("Routerlicious", () => {
+describe("someObjectlicious", () => {
```

Comment:
lol awesome

Reply:
lol didn't see this happen when I did find and replace